app/src/renderer/vuex/modules/delegates.js

```diff
@@ -15,8 +15,12 @@ export default ({ node }) => {
     setDelegateLoading(state, loading) {
       state.loading = loading
     },
-    setDelegates(state, delegates) {
-      state.delegates = delegates
+    setDelegates(state, validators) {
+      validators.forEach(validator => {
+        validator.id = validator.owner
```

Comment:
is there any reason we can't just use `validator.owner`?

Reply:
another way of asking this question: do we need `validator.id` for something?

Reply:
I don't think we need validator.id anymore. It was a way to secure that a fixed property on the validator holds something like an id. We have the owner now which should never change.